packages/functional-tests/tests/key-stretching-v2/changePassword.spec.ts

```diff
@@ -21,9 +21,12 @@ test.describe('severity-2 #smoke', () => {
   ) {
     const client = target.createAuthClient(version);
     const response = await client.signIn(email, password, { keys: true });
+    expect(response.keyFetchToken).toBeDefined();
+    expect(response.unwrapBKey).toBeDefined();
+
     const keys = client.accountKeys(
-      response.keyFetchToken,
-      response.unwrapBKey
+      response.keyFetchToken as string,
```

Comment:
Huh, why are these casts needed across these few files? It looks like `response` is type `SignedInAccountData`...

Reply:
It's also possible it's undefined. I don't like these casts either, and it seemed rendundant to do this with a type guard. Furthermore if I used a type guard, then I'd get a linting error to 'avoid conditionals' in tests.